docs/.eleventy/layouts/base.njk

```diff
@@ -475,12 +472,11 @@ img.company-logo {
 #toc-drawer-button {
   display: none;
 }
-@media (min-width: 1110px) {
-  .markdown-body table {
-    width: 66vw;
-  }
+#background-video {
+  image-rendering: crisp-edges;
```

Comment:
Couldn’t resist 😉

Reply:
Yeah, seems to be an image property

Reply:
It makes a difference in Firefox, but you’re right that it doesn’t seem to do anything for Chrome. :/